confluence_test.go

```diff
@@ -22,7 +22,7 @@ func doTest(t *testing.T, tdt []testData) {
 		output := string(renderer.Render(ast))
 
 		if output != v.expected {
-			t.Errorf("got:%v\nwant:%v", output, v.expected)
+			t.Errorf("got:%#v\nwant:%#v", output, v.expected)
```

Comment:
I used `"#v"` to be able to see all the new line characters in the strings. It helped me with writing the test case but I can revert it if you want.